project/src/main/java/edu/depaul/cdm/se452/project/model/Student.java

```diff
@@ -36,17 +36,37 @@
 	private int gpa;
 	
 	//@Column
-	private String firstName;
+	private String first_name;
 	
 	//@Column
-	private String lastName;
+	private String last_name;
 	
 	//@Column
 	private String email;
+	
+	public int getStudent_id() {

```

Comment:
Getters/Setters should not be needed since we're using Lombok.

Reply:
i'm getting nothing from mongo without them but if it works on your end then forget them